src/main/kotlin/com/nextstep/jngcii/lotto/model/Lotto.kt

```diff
@@ -0,0 +1,55 @@
+package com.nextstep.jngcii.lotto.model
+
+class Lotto(_numbers: List<Int>) {
+    val numbers: List<Int>
+
+    init {
+        validateCount(_numbers)
+        validateDuplication(_numbers)
+        validateRange(_numbers)
+
+        numbers = _numbers.sorted()
+    }
+
+    fun getSameCount(comparison: Lotto): Int {
+        var sameCount = INITIAL_COUNT
+
+        numbers.forEach {
+            sameCount += it.isIn(comparison.numbers)
+        }
+
+        return sameCount
+    }
```

Comment:
```suggestion
    fun getSameCount(comparison: Lotto): Int {
            return numbers.intersect(comparison.numbers.toSet()).count()
    }
```

이렇게 한번에 표현이 가능할 것 같습니다 ″̮

https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.math/truncate.html

Reply:
[refactor : Lotto의 numbers 타입을 Set으로 변경 및 sameCount 로직 리팩터링](https://github.com/next-step/kotlin-lotto/pull/333/commits/f1b2cd444ebeeab62e7189422bfe75a872f67b25)
반영했습니다! :)